requirements/dev.txt

```diff
@@ -5,87 +5,87 @@
 #    make upgrade
 #
 appdirs==1.4.4            # via -r requirements/quality.txt, -r requirements/travis.txt, fs, virtualenv
-argparse==1.4.0           # via -r requirements/quality.txt, caniusepython3
 astroid==2.3.3            # via -r requirements/quality.txt, pylint, pylint-celery
-attrs==20.2.0             # via -r requirements/quality.txt, pytest
-backports.functools-lru-cache==1.6.1  # via -r requirements/quality.txt, caniusepython3
-caniusepython3==7.2.0     # via -r requirements/quality.txt
-certifi==2020.6.20        # via -r requirements/quality.txt, -r requirements/travis.txt, requests
-chardet==3.0.4            # via -r requirements/quality.txt, -r requirements/travis.txt, requests
+attrs==21.2.0             # via -r requirements/quality.txt, edx-ace, pytest
+backports.functools-lru-cache==1.6.4  # via -r requirements/quality.txt, caniusepython3
+caniusepython3==7.3.0     # via -r requirements/quality.txt
+certifi==2020.12.5        # via -r requirements/quality.txt, -r requirements/travis.txt, requests
+chardet==4.0.0            # via -r requirements/quality.txt, -r requirements/travis.txt, diff-cover, requests
 click-log==0.3.2          # via -r requirements/quality.txt, edx-lint
 click==7.1.2              # via -r requirements/pip-tools.txt, -r requirements/quality.txt, click-log, code-annotations, edx-lint, pip-tools
-code-annotations==0.7.0   # via -r requirements/quality.txt
-codecov==2.1.9            # via -r requirements/travis.txt
-coverage==5.3             # via -r requirements/quality.txt, -r requirements/travis.txt, codecov, pytest-cov
-ddt==1.4.1                # via -r requirements/quality.txt
-diff-cover==4.0.0         # via -r requirements/dev.in
+code-annotations==0.10.2  # via -r requirements/quality.txt
+codecov==2.1.11           # via -r requirements/travis.txt
+coverage[toml]==5.5       # via -r requirements/quality.txt, -r requirements/travis.txt, codecov, pytest-cov
+ddt==1.4.2                # via -r requirements/quality.txt
+diff-cover==4.2.3         # via -r requirements/dev.in
 distlib==0.3.1            # via -r requirements/quality.txt, -r requirements/travis.txt, caniusepython3, virtualenv
-django==2.2               # via -c requirements/constraints.txt, -r requirements/quality.txt, code-annotations, djangorestframework, edx-i18n-tools
-djangorestframework==3.11.1  # via -r requirements/quality.txt
+django==2.2               # via -c requirements/constraints.txt, -r requirements/quality.txt, code-annotations, djangorestframework, edx-ace, edx-i18n-tools
+djangorestframework==3.12.4  # via -r requirements/quality.txt
+edx-ace==0.1.17           # via -r requirements/quality.txt
```

Comment:
@arjunsinghy96 `edx-ace` should be in `base.in` right? But I don't see that change. Have I missed it?

Reply:
@shimulch I have kept edx-ace in test.in as LMS will already have edx-ace and these two dependencies might conflict. (Just to be on the safe side)
For this project we need edx-ace primarily for testing as the application will not be run stand-alone.

However, please let me know if keeping it in base.in makes more sense.

Reply:
@arjunsinghy96 We should keep our core dependencies on the `base.in`. Check the [file](https://github.com/open-craft/custom-student-verification/blob/main/requirements/base.in). If there were any issue with dependency conflict then `django`, `djangorestframework` these dependencies would not be there. Keep `edx-ace` in the `base.in` as same as `django`. :)

Reply:
Thanks for the update @shimulch . I have moved `edx-ace` to `base.in`.